api/app/Models/Timezones.php

```diff
@@ -0,0 +1,53 @@
+<?php namespace App\Models;
+
+use Illuminate\Support\Collection;
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * Class Timezones
+ * @package App\Models
+ *
+ * Note this model does not have an associated database table as it is an abstract
+ * data model with generated token data.
+ */
+class Timezones extends Model
```

Comment:
Yes, I'm thinking something along those lines as well. As it is not a data model to interact with, just a list of data to retrieve. It doesn't make complete sense to use a model for timezones, but could go as a repo/service/helper?